packages/demo-app/src/App.tsx

```diff
@@ -6,9 +6,11 @@ import { MyBalance } from './MyBalance';
 
 const DEVNET_NODE_TM_URL = 'http://devnet.regen.network:26657'; // CORS not enabled on this one yet.
 const AMAURY_NODE_TM_URL = 'http://18.223.151.187:26657';
```

Comment:
Feel free to remove all AMAURY_ stuff! They were ec2 instances that don't exist anymore

Reply:
done